docs/wiki/guides/current/SoftSerial.md

```diff
@@ -0,0 +1,254 @@
+# SoftSerial
+
+Softserial provides a means to add another 'virtual' UART on MPU's with a limited number of pinned-out true UART ports.
```

Comment:
```suggestion
Softserial provides a means to add another 'virtual' UART on MCU's with a limited number of pinned-out true UART ports.
```

Also would think off we only need SOFTSERIAL on F411 and perhaps F405.

Reply:
Thanks for noting MPU.... fixed MPU to MCU or FC as appropriate.
Mostly softserial is needed on F411, but it really depends on what UARTS are pinned out, as much as the capability of the MCU.  Added a note, `eg F411's.`